src/test/ui/coherence/impl[t]-foreign[foreign[t],local]-for-foreign.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(re_rebalance_coherence)]
+
+// run-pass
```

Comment:
Can you get if the tests in the PR need to be run-pass? Seems check-pass is sufficient?

Reply:
yep fixed